react/RuiExpandCollapseReact/src/IRuiExpandCollapse.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Deloitte Digital 2019
+ *
+ * This source code is licensed under the BSD-3-Clause license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+declare global {
+	namespace JSX {
+		interface IntrinsicElements { // eslint-disable-line @typescript-eslint/interface-name-prefix
+			'rui-expand-collapse': any;
+		}
+	}
+}
+
+export interface IRuiExpandCollapseProps {
+	onChange?: Function,
+    open?: boolean,
+    children?: JSX.Element[] | JSX.Element;
```

Comment:
Can you change for `React.ReactNode` please

Reply:
Also lets remove the `?` from children. We want it to complain if it doesn't receive any.

Reply:
@jennasalau fixed